.goreleaser.yml

```diff
@@ -12,7 +12,7 @@ before:
     - >- # On linux the completions are used in nfpms below, but on macos they are used outside in the deployment build.
       {{ if eq .Runtime.Goos "windows" }}echo{{ end }} make completions
     - >- # We need to create the `.syso` files (per architecture) to embed Windows resources (version info)
-      pwsh .\script\gen-winres.ps1 386,amd64,arm64 '{{ .Version }}' .\script\winres.json .\cmd\gh\
+      {{ if ne .Runtime.Goos "windows" }}echo{{ end }} pwsh .\script\gen-winres.ps1 '{{ .Version }} ({{time "2006-01-02"}})' '{{ .Version }}' .\script\versioninfo.template.json .\cmd\gh\
```

Comment:
question: Is the current timestamp meant to take the place of the `Build` portion of the file version?